### ISSUE TITLE
Naphtaali tiles render as blank

### ISSUE THREAD

**Issue by syntaxaire** [TrashMonks/hagadias#21]
![image](https://user-images.githubusercontent.com/2661404/94490817-ff115880-01b4-11eb-956f-c54b93d37071.png)

**Comment by egocarib** [comment]
FYI - this also affects `Glowfish`.

I am not sure what's up because these tiles all render okay in QBE. Perhaps something discord doesn't like about them? 🤔

**Comment by egocarib** [comment]
@syntaxaire Is it possible these texture files are missing on Crypto's server? It works on my local crypto instance.

![image](https://user-images.githubusercontent.com/3429497/107864706-81c54800-6e24-11eb-8b46-a8912e63f561.png)

**Comment by syntaxaire** [comment]
Looks like this is caused by a backslash in the tile path for Glowfish in `ObjectBlueprints.xml`.

**Comment by syntaxaire** [comment]
Here's the complete set of problem tiles on Linux:
```
WARNING:root:Couldn't render tile for Starch: Items/sw_Jam.bmp not found at Textures/Items/sw_Jam.bmp
WARNING:root:Couldn't render tile for Lignin: Items/sw_Jam.bmp not found at Textures/Items/sw_Jam.bmp
WARNING:root:Couldn't render tile for Starapple Preserves: Items/sw_Jam.bmp not found at Textures/Items/sw_Jam.bmp
WARNING:root:Couldn't render tile for Spine Fruit Jam: Items/sw_Jam.bmp not found at Textures/Items/sw_Jam.bmp
WARNING:root:Couldn't render tile for Brinestalk: Tiles/tile-brinestalk.png not found at Textures/Tiles/tile-brinestalk.png
WARNING:root:Couldn't render tile for Glowfish: Creatures\sw_glowfish.bmp not found at Textures/Creatures\sw_glowfish.bmp
WARNING:root:Couldn't render tile for Ghost Perch: Creatures\sw_glowfish.bmp not found at Textures/Creatures\sw_glowfish.bmp
WARNING:root:Couldn't render tile for Naphtaali Forager: Creatures/sw_naphtaali_forager.bmp not found at Textures/Creatures/sw_naphtaali_forager.bmp
WARNING:root:Couldn't render tile for Naphtaali Runt: Creatures/sw_naphtaali_runt.bmp not found at Textures/Creatures/sw_naphtaali_runt.bmp
WARNING:root:Couldn't render tile for Naphtaali Blowgunner: Creatures/sw_naphtaali_blowgunner.bmp not found at Textures/Creatures/sw_naphtaali_blowgunner.bmp
WARNING:root:Couldn't render tile for Naphtaali Stalker: Creatures/sw_naphtaali_stalker.bmp not found at Textures/Creatures/sw_naphtaali_stalker.bmp
WARNING:root:Couldn't render tile for BetaPlant: Creatures/creatures-seedspittingvine.png not found at Textures/Creatures/creatures-seedspittingvine.png
WARNING:root:Couldn't render tile for MarbleFloor: Tiles/tile-dirt.png not found at Textures/Tiles/tile-dirt.png
```

**Comment by syntaxaire** [comment]
The case for Glowfish is fixed in d150523 (fixed '\' handling). The other problems are all caused by case sensitivity on Linux.

**Comment by egocarib** [comment]
Closing this issue to consolidate tracking. The remaining problem is tracked in #37
